src/js/components/LoginPage/Login.js

```diff
@@ -1,11 +1,100 @@
 import React, { Component } from "react";
 import "./Login.scss";
 import "./ButtonStyles.css";
+import Navbar from "../Navbar/Navbar";
+import {Redirect} from "react-router-dom";
+import StyledFirebaseAuth from "../RegisterPage/SignUp";
+import {RegisterUser} from "../../services/apiservice";
+import { BrowserRouter, Route, Switch, NavLink } from "react-router-dom";
+
+
 class Login extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      name: '',
+      surname: '',
+      email: '',
+      phone: '',
+      password: '',
+      organisation: '',
+
+    };
+
+    // this.handleClick = this.handleClick.bind(this);
+  }
+
+  updateNameValue(evt) {
+    this.setState({name: evt.target.value});
+  }
+
+  updateSurnameValue(evt) {
+    this.setState({surname: evt.target.value});
+  }
+
+  updateEmailValue(evt) {
+    this.setState({email: evt.target.value});
+  }
+
+  updatePhoneValue(evt) {
+    this.setState({phone: evt.target.value});
+  }
+
+  updatePasswordValue(evt) {
+    this.setState({password: evt.target.value});
+  }
```

Comment:
add empty line between declaration. remove ghost code

Reply:
Thanks Removed it